source/Minable.cpp

```diff
@@ -248,3 +255,26 @@ const int64_t &Minable::GetValue() const
 {
 	return value;
 }
+
+
+
+Minable::Payload::Payload(const DataNode& node)
```

Comment:
isnt it supposed to be at the same place in the cpp and h? or is it here bc its a subclass?

Reply:
It was originally private, but I made it public and never moved the implementation.